napari/_vispy/utils/gl.py

```diff
@@ -111,40 +111,46 @@ def fix_data_dtype(data):
         return data.astype(dtype)
 
 
+# blend_func parameters are multiplying:
+# - source color
+# - destination color
+# - source alpha
+# - destination alpha
+# they do not apply to min/max blending equation
+
 BLENDING_MODES = {
     'opaque': dict(
         depth_test=True,
         cull_face=False,
         blend=False,
-        blend_func=('one', 'zero'),
+        blend_func=('one', 'zero', 'one', 'one'),
```

Comment:
pretty sure from the GL docs that if `blend=false` as above, then the `blend_func` and `blend_equation` do nothing, are irrelevant, could be deleted.

Reply:
Oh yeah, of course!